lib/createRss.js

```diff
@@ -0,0 +1,128 @@
+'use strict';
+
+const _ = require('lodash');
+const ARRAY = [];
+const fs = require('fs-extra');
+const NUMBER = 0;
+const OBJECT = {};
+const path = require('path');
+const RSS = require('rss');
+const STRING = '';
+const url = require('url');
+
+const blogOptionTypes = {
+	title: STRING,
+	description: STRING,
+	url: STRING,
+	guid: STRING,
+	categories: ARRAY,
+	author: STRING,
+	date: STRING,
+	lat: NUMBER,
+	long: NUMBER,
+	custom_elements: ARRAY,
+	enclosure: OBJECT
+};
+
+const feedOptionTypes = {
+	categories: ARRAY,
+	copyright: STRING,
+	custom_elements: ARRAY,
+	custom_namespaces: OBJECT,
+	description: STRING,
+	docs: STRING,
+	feed_url: STRING,
+	generator: STRING,
+	hub: STRING,
+	image_url: STRING,
+	language: STRING,
+	managingEditor: STRING,
+	pubDate: STRING,
+	title: STRING,
+	ttl: NUMBER,
+	site_url: STRING,
+	webMaster: STRING
+};
+
+const createRSS = (data, opts) => {
```

Comment:
Let's not use arrow functions, we don't want to drop support for current LTS versions of Node.js that can't use them.